data/xml/2023.nejlt.xml

```diff
@@ -0,0 +1,244 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2023.nejlt">
+  <volume id="1" ingest-date="2024-08-06" type="proceedings">
+    <meta>
+      <booktitle>Northern European Journal of Language Technology, Volume 9</booktitle>
+      <editor><first>Leon</first><last>Derczynski</last></editor>
+      <publisher>Northern European Association of Language Technology</publisher>
+      <address>Copenhagen, Denmark</address>
```

Comment:
I realized that this is a mistake that also needs to be corrected in previous issues; it should be:

```xml
<publisher>Linköping University Electronic Press</publisher>
<address>Linköping, Sweden</address>
```

(I could open a PR for previous issues.)

Reply:
Do you want to just fix these and add them to this PR?